Uniswap-Swap-Event-Detection-Bot/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "challenge-2",
+  "version": "0.0.1",
+  "description": "Forta bot which detects swaps made on uniswapv3 without keeping a reference of pools",
+  "chainIds": [
+    1
+  ],
+  "repository": "https://github.com/gautam2002/forta-bots-challenges/Uniswap-Swap-Event-Detection-Bot",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "info": "forta-agent info",
+    "logs": "forta-agent logs",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
```

```suggestion
    "test": "jest --detectOpenHandles"
```
added this because jest returns a warning about it